tools/value-generation/gen.py

```diff
@@ -134,6 +145,9 @@ def processCluster(crd_value, value_map, comment_map) :
   # Set this empty to ensure we auto-generate it by default
   value_map[crd_value]['security']['adminSecret'] = ''
 
+  # Set monitoring to emtpy since it requires image which is not provided by default
+  value_map[crd_value]['monitoring'] = {}
```

Comment:
We should probably sort that soon

Reply:
Thinking the operator should provide default image?

Reply:
Not part of this task though.

Reply:
Agree